### PR TITLE
ZEN-17164: Redesign FileDropzone

### PR DESCRIPTION
**Pull request by chavlji**
As requested by Igor, I've changed the design of FileDropzone.
Check jira:
https://reciprocitylabs.atlassian.net/browse/ZEN-17164

### CODE REVIEW

**Comment by zenbot-0** [comment]
🔬 [Click here](https://reciprocity.github.io/zenlib/branches/17164-Filedropzone-redesign) to test your changes on Storybook

**Review by mplesec — COMMENTED** [review]
Is it intentional that it's missing the cloud picture?

**Comment by chavlji** [comment]
@mplesec it shouldn't 🤔 
Screenshot 2020-09-25 at 13 48 27

File: packages/file-dropzone/FileDropzone.vue [hunk 1]
```diff
@@ -84,14 +85,16 @@ export default {
 <style lang="scss" scoped>
 @import "@reciprocity/bootstrap/scss/colors";
 @import "@reciprocity/bootstrap/scss/variables";
+@import "../_styles_/scss/colors";
```
Comment (mplesec) [inline packages/file-dropzone/FileDropzone.vue:88]: Shouldn't this be `@import "@reciprocity/styles/scss/colors.scss"`?

**Comment by mplesec** [comment]
This is what it looks like on the storybook link in the PR:
Screenshot 2020-09-25 at 13 51 16

**Review by mplesec — APPROVED** [review]

**Comment by leferreyra** [comment]
Yeah, @chavlji you shouldn't be developing components in ZenGRC. The idea is that you do it in Storybook. Linking is for very specific situations only.

**Comment by chavlji** [comment]
@leferreyra I know. It's just that I couldn't get Storybook to pick up my changes... I've tried everything. Even deleted all  node_modules. I'll call you up to see what I'm doing wrong and maybe update docs.

**Comment by leferreyra** [comment]
@chavlji Ok. What I'm saying is that if it doesn't work on Storybook. It doesn't work. And we shouldn't have merged it.
No big deal tho. We only use this in zen atm. But something to have in mind for next time.

**Pull request merged**